src/pages/basic_data/institution_admin/FindPersonPanel.tsx

```diff
@@ -50,7 +50,7 @@ export const FindPersonPanel = () => {
               </Button>
             ) : (
               <>
-                <Typography variant="h3" sx={{ mt: '1rem' }}>
+                <Typography component="h2" variant="h3" sx={{ mt: '1rem' }}>
```

Comment:
Jeg tenkte h3 var mest naturlig, siden denne er under `<h2>Person</h2>` 🤔 

![bilde](https://github.com/user-attachments/assets/c432b574-e34c-4a7d-b480-65a84ded92b4)

```suggestion
                <Typography variant="h3" sx={{ mt: '1rem' }}>
```